common/rfb/CConnection.cxx

```diff
@@ -49,7 +49,7 @@ static LogWriter vlog("CConnection");
 CConnection::CConnection()
   : csecurity(0),
     supportsLocalCursor(false), supportsCursorPosition(false),
-    supportsDesktopResize(false), supportsLEDState(false),
+    supportsDesktopResize(false), supportsLEDState(false), supportsQEMUAudio(false),
```

Comment:
We try to use generic phrasing for this, so future extensions can also be handled. So please rename this to something like `supportsAudio`.

Reply:
OK